Trackovid19-web/src/app/screens/post-code/post-code.component.html

```diff
@@ -75,7 +71,7 @@
         <input
           class="frist-type-number"
           formControlName="zip-code-1"
-          type="text"
+          type="number"
           pattern="[0-9]*"
```

Comment:
Can you add a validation to have 4 number? It should be possible to send 3 and now we can do it

Reply:
Done